### PR TITLE
Cleanup of HTML syntax in the Producers.html file

### PR DESCRIPTION
**Pull request by teolemon**
Cleanup of HTML syntax

### CODE REVIEW

File: producers.html [hunk 1]
```diff
@@ -1,370 +1,343 @@
...
-  
-<h3 id="exemples_de_producteurs_qui_transmettent_les_photos_et_donnees_de_leurs_produits_a_open_food_facts">Examples of producers who submit photos and product data to Open Food Facts</h3>
+        <li>The data is then available in more than 100 applications.</li>
 
-<p>A growing number of high level French producers are sending us the data of their products: Fleury Michon, Sodebo, System U, Carrefour, Casino, Franprix, Monoprix.</p>
+        <li>You can easily obtain statistics about your products and export the data of your products supplemented by indicators calculated by Open Food Facts such as the Nutri-Score.</li>
+    </ul>
 
-<h2 id="comment_ajouter_les_donnees_de_ses_produits_dans_open_food_facts">How to add the data of your products in Open Food Facts</h2>
+    <h3 id="exemples_de_producteurs_qui_transmettent_les_photos_et_donnees_de_leurs_produits_a_open_food_facts">Examples of producers who submit photos and product data to Open Food Facts</h3>
```
Comment (VaiTon) [inline producers.html]: What a long id! :P

File: producers.html [hunk 2]
```diff
@@ -1,370 +1,343 @@
...
-  
-<h3 id="quelles_donnees">What data</h3>
+    <p>A growing number of high level French producers are sending us the data of their products: Fleury Michon, Sodebo, System U, Carrefour, Casino, Franprix, Monoprix.</p>
 
-<p>All the data you have can be integrated into Open Food Facts: ingredients and nutritional information of course, but also labels, materials used for packaging, traceability of ingredients, instructions for preparation and recycling, ecological footprint etc.</p>
-<p>We regularly add new fields to the database based on the information on the packages or provided by the manufacturers.</p>
-We have recently added fields on the carbon footprint, and the packaging of products, which are already highlighted on Open Food Facts and in applications that help to change the behavior in terms of recycling.<p></p>
+    <h2 id="comment_ajouter_les_donnees_de_ses_produits_dans_open_food_facts">How to add the data of your products in Open Food Facts</h2>
```
Comment (VaiTon) [inline producers.html]: Same as before, can't we change this id?

File: producers.html [hunk 3]
```diff
@@ -1,370 +1,343 @@
...
+        <li>glycemic-index_100g</li>
+        <li>Waterpolo hardness_100g</li>
+        <li>choline_100g</li>
+        <li>phylloquinone_100g</li>
+        <li>beta-glucan_100g</li>
+        <li>inositol_100g</li>
+        <li>carnitine_100g</li>
+    </ul>
+
+    <h4 id="envoi_des_photos_et_des_donnees">Sending photos and data</h4>
```
Comment (VaiTon) [inline producers.html]: And I think IDs should be in english.

File: producers.html [hunk 4]
```diff
@@ -1,370 +1,343 @@
...
+
+    <p>When a product disappears from the shelves, it is quickly consumed, and we find that it is quickly no longer scanned.</p>
+    <p>We then archive the product so that it remains available as an archive for scientific purposes in the context of food cohorts that can last for decades. It is indeed essential to be able to obtain the compositions of the product at a precise date, even if it has changed or disappeared.</p>
+
+    <p>If you have historical data on your old products or old versions of your products, we would also be very interested in this historical data, in order to be able to go back in time, to allow scientific analyzes on long time, and to make it possible to visualize the evolution some products.</p>
+
+    <p>Producers can also send us the lists of barcodes removed from the sale so that we can label the products appropriately.</p>
+
+    <h3 id="est_ce_que_les_produits_pour_animaux_les_produits_non_alimentaires_les_cosmetiques_et_les_autres_produits_peuvent_etre_integres_a_open_food_facts">Can animal products, non-food products, cosmetics and other products be integrated into Open Food Facts?</h3>
```
Comment (VaiTon) [inline producers.html]: Long id

**Comment by Simon-Shapiro** [comment]
I'd like to work on this if nobody's started on it yet. I speak English and French, do you want the ID's to stay in French or should I change them to English?

**Comment by teolemon** [comment]
@Simon-Shapiro we should change them to English. We'll keep the French ones in the French translation for backwards compatibility

**Pull request closed without merging**